### ISSUE TITLE
Python inference gives always same result on quantized model regardless of input

### ISSUE THREAD

**Issue by dkwiatkow** [openvinotoolkit/openvino#4317]
Hi,
i am having a trouble with inference using python based API. When I try to execute following code:
```
ie = IECore()  
ie.add_extension(args.cpu_extension, "CPU")
net = ie.read_network(model=model)
net.batch_size = 1
exec_net = ie.load_network(network=net, device_name="CPU", num_requests=0)
exec_net.infer(inputs={'0': data})
```
I get result in the correct shape, but whatever input I pass, I get the same result. If I pass prequantized IR model I get correct behavior so I guess there is problem in quantization. My quantization config file I use:
```
{
    /* Model parameters */

    "model": {
        "model_name": "fixed_input_model",
        "model": "/home/dkwiatko/openvino/vino_model.xml",
        "weights": "/home/dkwiatko/openvino/vino_model.bin"
    },

    "engine": {
        "name": "vino_model",
        "launchers": [
            {
                "framework": "dlsdk",
                "device": "CPU",
                "adapter": "classification"
            }
        ],
        "datasets": [
            {
                "name": "dataset_name",
                "data_source": "<path to bunch of wav files>",
                "reader": {
                    "type": "wav_reader"
                },
                "preprocessing": [
                    {
                        "type": "clip_audio",
                        "duration": "49152 samples",
                    }
                ],
                "metrics": [
                    {
                        "name": "accuracy@top1",
                        "type": "accuracy",
                        "top_k": 1
                    }
                ]
            }
        ],
    },
    "compression": {
        "target_device": "CPU", 
        "algorithms": [
            {
                "name": "DefaultQuantization", 
                "params": {
                    "preset": "accuracy",
                    "stat_subset_size": 200
                }
            }
        ]
    }
}
``` 
How could I fix this behavior? I tried many hyperparameters but still getting same error

**Comment by ilya-lavrenov** [comment]
CC @dmitry-gorokhov @AlexKoff88

**Comment by Iffa-Meah** [comment]
Hi,

Fyi, this thread is a duplicate to this one [here](https://community.intel.com/t5/Intel-Distribution-of-OpenVINO/Python-inference-gives-always-same-result-on-quantized-model/m-p/1259286#M22949).
Please help to refer to the solution there.

Hope it helps!


Sincerely,
Iffa

**Comment by AlexKoff88** [comment]
@Iffa-Meah, can you please clarify what you mean by "I get the same result". Is the performance the same? 
Can you please also provide information about OV version, DL model architecture, target hardware?

**Comment by Iffa-Meah** [comment]
@AlexKoff88 , I didn't get the same result,instead **it's the customer** that keep getting the same result whenever he feeds on different input to his Quantized mode.

Hence, I compared the performance between original model, FP32 model and Quantized model to poc the theory of optimized performance by performing quantization.

Next, I feed 2 different input to the quantized model and I can see the different result from it.
He probably need to change his config file according to his model, which I had attached in the thread as a reference.

I used:
OpenVino 2021.1
Model I used mobilenet-v2-pytorch as a test subject since the customer didn't mention his
Target hardware: CPU

**Comment by AlexKoff88** [comment]
Ok. @dkwiatkow, does it mean that you prepare two different data inputs and get the same results for the quantized model while FP32 model produces different results?
If it does, how about the accuracy after quantization? Is it still close to the original model?
I have 3 hypotheses about what is wrong with the model:
- Something wrong with the data/tensors that are being fed to the model
- Accuracy of the quantized model is poor so that it produces incorrect results
- Problems in low-precision runtime

So we need to localize the problem. @dkwiatkow, is it possible to share the model and dataset?

**Comment by Iffa-Meah** [comment]
Closing issue, feel free to re-open or start a new issue if additional assistance is needed.
